<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenTextDialog</class>
 <widget class="QDialog" name="OpenTextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="textCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of lines to display per screen:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="numberOfLinesSpinner">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Jump to line:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="goToLineSpinner">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="loadEntireText">
     <property name="text">
      <string>Load entire text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenTextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenTextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadEntireText</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberOfLinesSpinner</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadEntireText</sender>
   <signal>toggled(bool)</signal>
   <receiver>goToLineSpinner</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
